jekyll/_cci2/configuration-reference.md

```diff
@@ -465,8 +465,8 @@ The `orbs` key is supported in `version: 2.1` configuration
 Key | Required | Type | Description
 ----|-----------|------|------------
 orbs | N | Map | A map of user-selected names to either: orb references (strings) or orb definitions (maps). Orb definitions must be the orb-relevant subset of 2.1 config. See the [Creating Orbs]({{ site.baseurl }}/creating-orbs/) documentation for details.
-executors | N | Map | A map of strings to executor definitions. See the [Executors]({{ site.baseurl }}/configuration-reference/#executors-requires-version-21) section below.
-commands | N | Map | A map of command names to command definitions. See the [Commands]({{ site.baseurl }}/configuration-reference/#commands-requires-version-21) section below.
+executors | N | Map | A map of strings to executor definitions. See the [Executors]({{ site.baseurl }}/configuration-reference/#executors) section below.
```

Comment:
Not sure why this is in here, I thought this change was in master. Either way it's correct!